### PR TITLE
feat: Add placeholder Tiled map with collisions

### PR DESCRIPTION
**Pull request by DonLe27**
Used Tiled collision editor to make map. 
Matter physics is needed to draw non-square boundary boxes for objects (as opposed to AP)
Refactored index.js to make each scene a class

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,5 +26,9 @@
     "webpack-cli": "^4.5.0",
     "webpack-dev-server": "^3.11.2",
     "webpack-merge": "^5.7.3"
+  },
+  "dependencies": {
+    "http-server": "^0.12.3",
+    "style-loader": "^2.0.0"
```
Comment (BryanPan342) [inline package.json]: do we need these?

File: test.tmx [hunk 2]
```diff
@@ -0,0 +1,153 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<map version="1.4" tiledversion="1.4.1" orientation="orthogonal" renderorder="right-down" width="25" height="25" tilewidth="32" tileheight="32" infinite="0" nextlayerid="3" nextobjectid="1">
```
Comment (BryanPan342) [inline test.tmx]: can we add a more descriptive name for this?

File: tiled_project.tiled-session [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        "test.tmx",
+        "pokemon_tileset_ 4.tsx"
+    ],
+    "recentFiles": [
+        "pokemon_tileset_ 4.tsx",
+        "test.tmx",
+        "pokemon_tileset_cropped3.tsx",
+        "pokemon_tileset_cropped.tsx",
+        "pokemon_tileset_2.tsx"
+    ]
```
Comment (BryanPan342) [inline tiled_project.tiled-session]: not sure how some of these files are needed like the `.tsx` extension is needed

**Review by BryanPan342 — COMMENTED** [review]
some comments that i dont really undertand

File: package.json (hunk 1 above)
Comment (DonLe27) [reply package.json]: we need style-loader, i removed http-server tho

**Review by BryanPan342 — APPROVED** [review]
KING

**Pull request merged**